.flake8

```diff
@@ -1,2 +1,7 @@
 [flake8]
+exclude = .eggs,build,docs,.venv*
 ignore = E203 E501 W503 W504
+per-file-ignores =
+    praw/models/__init__.py:F401
```

Comment:
I would even go as far as to do
`praw/*/__init__.py`

Reply:
I think it's fine as is.